mappings/net/minecraft/util/collection/ImmutablePool.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_6005 net/minecraft/util/collection/ImmutablePool
```

Comment:
This isn't an immutable pool, but one pool easier to use by specializing to handle `Present` weighted entries.

Reply:
I don't know what else I would call it then. I called it this because it seems to follow the patterns of google's ImmutableList/Set/etc.

It has some `of` methods, as well as a `Builder` for constructing instances. There's no `put`, `set`, or `add` on this class either.